### PR TITLE
Reduce MAX_VECTOR_LENGTH

### PR DESCRIPTION
**Pull request by alexcombessie**
This branch has been created from the partitioning branch so it should be reviewed after the PR #5 

[ch59735](https://app.clubhouse.io/dataiku/story/59735/reduce-max-vector-length)

### CODE REVIEW

**Comment by alexcombessie** [comment]
- [ ] Rename vectors to arrays everywhere in the code

File: python-lib/data_loader.py [hunk 1]
```diff
@@ -39,37 +39,50 @@ def _validate_df(self, df: pd.DataFrame):
...
-        if verbose:
-            logging.info(
-                f"Loading dataframe into vector format: array of dimensions {vectors.shape} "
-                + f"loaded in {perf_counter() - start:.2f} seconds.",
-            )
-        return (vector_ids, vectors)
+        arrays = np.ascontiguousarray(arrays[:, :i], dtype=np.float32)
+        return arrays
+
+    def _count_array_length(self, df: pd.DataFrame) -> int:
```
Comment (tdesfont) [inline python-lib/data_loader.py:83]: For the function _count_array_length, what is the first array here, is it not supposed to be the only one we get ?

Comment (tdesfont) [reply python-lib/data_loader.py:83]: Ah ok this it the first row of dataset

**Review by tdesfont — APPROVED** [review]
Tested on local project, result is OK

**Pull request merged**